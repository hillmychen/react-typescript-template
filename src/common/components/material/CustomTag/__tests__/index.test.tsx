import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CustomTag } from '../index'

it('CustomTag title', () => {
  const title = '标题'
  const { asFragment, getByText } = render(<CustomTag color="blue">{title}</CustomTag>)
  expect(getByText(title)).toBeDefined()
  expect(asFragment()).toMatchSnapshot()
})

it('CustomTag green', () => {
  const title = '标题'
  const { asFragment, getByText } = render(<CustomTag color="green">{title}</CustomTag>)
  expect(getByText(title)).toBeDefined()
  expect(asFragment()).toMatchSnapshot()
})
