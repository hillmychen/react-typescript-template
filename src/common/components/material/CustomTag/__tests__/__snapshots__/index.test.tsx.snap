// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTag green 1`] = `
<DocumentFragment>
  <span
    class="customTag green"
  >
    标题
  </span>
</DocumentFragment>
`;

exports[`CustomTag title 1`] = `
<DocumentFragment>
  <span
    class="customTag blue"
  >
    标题
  </span>
</DocumentFragment>
`;
